packages/bff/src/services/catalogService.ts

```diff
@@ -122,5 +124,33 @@ export function catalogServiceBuilder(
 
       return response;
     },
+    createEService: async (
+      eServiceSeed: bffApi.EServiceSeed,
+      { headers }: WithLogger<BffAppContext>
+    ): Promise<CreatedResource> =>
+      await catalogProcessClient.createEService(eServiceSeed, {
```

Comment:
These routes should return only the `CreatedResource` which is only the `id`

Reply:
Right now the definition in typescript is [with](https://zod.dev/?id=passthrough) `passthrough` which means it also accepts other fields and this is the reason why I didn't filter it. I think, for code readability, it is better to return everything but let, if necessary, `CreatedResource`, strip unneeded fields.